### ISSUE TITLE
opera and ie support

### ISSUE THREAD

**Issue by paulirish** [maccman/gfx#3]
they support 2d transforms...

and thus GFX should support them :)

**Comment by mynameistechno** [comment]
Agreed! I like the idea of using jQuery's animation queue for CSS transforms. If I start using this I'll try to contribute back to opera and ie support.

**Comment by hlfcoding** [comment]
Related issue: for IE9 and below, is there a way for `$.fn.gfx` to defer to `$.fn.animate`? There should be.
